packages/react-accounts/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "meteor-react-accounts",
+  "scripts": {
+    "make-types": "npx typescript *.ts --declaration --emitDeclarationOnly --outDir types"
+  },
+  "dependencies": {
+    "@testing-library/react": "^10.0.2",
```

Comment:
Shouldn't `@testing-library/react` be in dev dependencies? The same with `react-test-renderer`.

Reply:
The entire package.json is a dev dependency, and has no impact on the final product (in fact, it's important not to publish with the `node_modules` folder present). I could still change it though.